components/BottomNavigation/src/private/MDCBottomNavigationItemView.m

```diff
@@ -577,6 +584,16 @@ - (void)setTitlePositionAdjustment:(UIOffset)titlePositionAdjustment {
   }
 }
 
+- (void)setTitleNumberOfLines:(NSInteger)titleNumberOfLines {
+  _titleNumberOfLines = titleNumberOfLines;
+  self.label.numberOfLines = self.titleBelowIcon ? titleNumberOfLines : 1;
```

Comment:
there would be cases the numberOfLines for the label would not be in sync with the titleNumberOfLines iVar. Is this the wanted behavior? If so, can you please clarify this in documentation. Lastly, why use a backing store to begin with and not just use self.label.numberOfLines?

Reply:
Yes, the expected behavior would be for `titlesNumberOfLines` to apply only when the title and icon are vertically "stacked". That is the only example in the Material Guidelines article showing multi-line titles.  I improved the header docs (it was missing in the public property).  

I felt that the combination of tests captured the behavior, both snapshot and unit tests.  Do you think there is something more that can be done to make the behavior more explicit/obvious?

Reply:
I forgot to answer the last question.  Because the value of `numberOfLines` on the UILabel is adjusted depending on the layout, a separate variable is required to store the user data.